views/schedule-form-templates.twig

```diff
@@ -135,8 +135,8 @@
         <div class="col-sm-2">
             <label for="criteria_type[]">
                 <select class="form-control" name="criteria_type[]">
-                    <option value="Custom" {{#eq type "Custom"}}selected{{/eq}}>{% endverbatim %}{% trans "Custom" %}{% verbatim %}</option>
-                    <option value="Weather" {{#eq type "Weather"}}selected{{/eq}}>{% endverbatim %}{% trans "Weather" %}{% verbatim %}</option>
+                    <option value="custom" {{#eq type "Custom"}}selected{{/eq}}>{% endverbatim %}{% trans "Custom" %}{% verbatim %}</option>
```

Comment:
Should we change the handlebars condition to be lowercase `custom` and `weather` in this case? 🤔